jerry-core/vm/vm.c

```diff
@@ -1393,8 +1393,16 @@ vm_loop (vm_frame_ctx_t *frame_ctx_p) /**< frame context */
 
           ecma_object_t *child_class_p = ecma_get_object_from_value (child_value);
           ecma_object_t *child_prototype_class_p = ecma_get_object_from_value (child_prototype_value);
-          ecma_property_value_t *prop_value_p;
 
+          ecma_property_t *property_p = ecma_find_named_property (child_prototype_class_p,
+                                                                  ecma_get_magic_string (LIT_MAGIC_STRING_CONSTRUCTOR));
+          if (property_p != NULL)
```

Comment:
If the constructor property is set in `VM_OC_PUSH_CLASS_CONSTRUCTOR_AND_PROTOTYPE` is it necessary to delete it? After this patch, I think the delete/create operations become unnecessary here.

Reply:
Yes, it's unnecessary here. I've remove it.